chore: add trace event (#6)
api7/cloud-go-sdk: README.md

# cloud-go-sdk

The Go SDK for accessing [API7 Cloud API](https://api7.cloud/api)

# Installation

> Make sure you're using Go Modules to manage your Go Project, and your Go version is larger than or equal to `1.19`.

Reviewer: ```suggestion
Make sure you're using Go Modules to manage your Go Project, and your Go version needs to be `1.19` or later.
```
